ddl/db_test.go

```diff
@@ -473,6 +474,67 @@ func (s *testDBSuite) TestCancelDropIndex(c *C) {
 	s.mustExec(c, "alter table t drop index idx_c2")
 }
 
+// TestCancelAddIndex1 tests canceling ddl job when the add index worker is not started.
+func (s *testDBSuite) TestCancelAddIndexPanic(c *C) {
+	gofail.Enable("github.com/pingcap/tidb/ddl/errorMockPanic", `return(true)`)
```

Comment:
Can be moved to fail_db_test.go

Reply:
It is not necessary.

Reply:
ddl.TestDDLCallback is defined in ddl/callback_test.go, and it is not located in the same directory in the fail_db_test.go, so we cannot move to `fail_db_test.go`, otherwise, it will reports `ddl/failtest/fail_db_test.go:395:11: undefined: ddl.TestDDLCallback`